src/main/java/org/fundacion/jala/converter/service/ObjectMetadata.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.service;
+
+import org.fundacion.jala.converter.service.metadata.TypeFileExport;
+
+import java.io.File;
+
+public class ObjectMetadata {
+    private File fileToExtract;
+    private File fileToExport;
+    private String nameExport;
+    private TypeFileExport typeFileExport = TypeFileExport.XMP;
+    private boolean moreInfo = false;
+
+    /**
```

Comment:
Java docs are missing in all methods of this class

Reply:
Solve